pkg/server/config.go

```diff
@@ -16,17 +16,36 @@ package server
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/abcxyz/pkg/cli"
 )
 
+// SupportedResourceScope defines the scopes that are supported
+// when doing resources searching.
+var SupportedResourceScope = map[string]struct{}{
+	"projects":      {},
+	"folders":       {},
+	"organizations": {},
+}
+
 // HandlerConfig defines the set over environment variables required
 // for running this application.
 type HandlerConfig struct {
 	Port           string `env:"PORT,default=8080"`
 	ProjectID      string `env:"PROJECT_ID,required"`
 	SuccessTopicID string `env:"PMAP_SUCCESS_TOPIC_ID,required"`
+	// FailureTopicID is optional for policy service
 	FailureTopicID string `env:"PMAP_FAILURE_TOPIC_ID"`
+	// MappingConfig is special config for mapping service
+	MappingConfig MappingConfig
```

Comment:
I think this should be the other way around. Define a `MappingHandlerConfig` that embeds `HandlerConfig`

Reply:
Done.